src/coffee-script.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
```

Comment:
Are there specs for this? I would imagine something similar to the babel/typescript ones